### PR TITLE
Replace Thrift types with TableNotFoundException

### PR DESCRIPTION
**Pull request by milleruntime**

### CODE REVIEW

File: src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java [hunk 1]
```diff
@@ -153,10 +150,8 @@ public void visit(State state, RandWalkEnv env, Properties props) throws Excepti
           env.getAccumuloClient().namespaceOperations().setProperty(namespace, property.getKey(),
               property.getDefaultValue());
         } catch (AccumuloException ex) {
-          if (ex.getCause() instanceof ThriftTableOperationException) {
-            ThriftTableOperationException ttoe = (ThriftTableOperationException) ex.getCause();
-            if (ttoe.type == TableOperationExceptionType.NAMESPACE_NOTFOUND)
-              return;
+          if (ex.getCause() instanceof TableNotFoundException) {
```
Comment (ctubbsii) [inline src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java]: Should this be NamespaceNotFoundException?

Comment (milleruntime) [reply src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java]: I think so good catch.  Fixed in 6a87136

File: src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java [hunk 2]
```diff
@@ -228,10 +222,8 @@ private void changeNamespaceSetting(RandomDataGenerator random, State state, Ran
       env.getAccumuloClient().namespaceOperations().setProperty(namespace,
           setting.property.getKey(), "" + newValue);
     } catch (AccumuloException ex) {
-      if (ex.getCause() instanceof ThriftTableOperationException) {
-        ThriftTableOperationException ttoe = (ThriftTableOperationException) ex.getCause();
-        if (ttoe.type == TableOperationExceptionType.NAMESPACE_NOTFOUND)
-          return;
+      if (ex.getCause() instanceof TableNotFoundException) {
```
Comment (ctubbsii) [inline src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java]: This one also seems to be looking for a NamespaceNotFoundException

Comment (milleruntime) [reply src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java]: Yup, nice catch.

**Pull request merged**